app/assets/stylesheets/base/globals.scss

```diff
@@ -0,0 +1,88 @@
+body {
+  font-family: 'Courier New', sans-serif;
+  font-size: 15px;
+  color: #190506;
+  margin: 15px;
+}
+
+nav ul {
+  display: flex;
+  justify-content: space-between;
+}
+
+a {
+  text-decoration: none;
+  color: #2E5077;
+}
+
+a:hover {
+  padding: 5px;
+  background-color: #4DA1A9;
+  color: #ffffff;
+  border-radius: 5px;
+}
+
+h1 {
+  font-size: 3.5em;
+  padding: 15px 0;
+}
+
+h2 {
+  font-size: 3em;
+  line-height: 2em;
+}
+
+h3 {
+  font-size: 1.5em;
+}
+
+p {
+  line-height: 1.5em;
+}
+
+.driver-card, .passenger-card {
+  display: grid;
+  grid-gap: 10px;
+  grid-template-columns: repeat(6, 220px);
```

Comment:
This makes the grid of passengers and drivers rather fixed in size, maybe doing it with an auto number of columns would be more responsive.